### ISSUE TITLE
Some airlocks are placed horizontally on vertical walls

### ISSUE THREAD

**Issue by PKPenguin321** [wall-nerds/wallening#13]
## Round ID:
182795

## Reproduction:

See centcomm:
![image](https://user-images.githubusercontent.com/8345184/167278843-b6df92db-1c2b-45b5-b1d7-17e3bc0fb482.png)
This causes you to only be able to see maybe 2/3rds of the sprite of the airlock as it for some reason renders underneath the top half of the wall below it, and is visually very confusing

**Comment by LemonInTheDark** [comment]
The core issue here is we're adding direction to airlocks, which they didn't used to have.
Essentially some poor mapper needs to go through and orient every single one.

We've not done it everywhere because that would just nuke us with conflicts, but it will need to happen eventually

**Comment by PKPenguin321** [comment]
Noted. Worth mentioning that this applies to other things as well, such as blast doors:
![image](https://user-images.githubusercontent.com/8345184/167279585-8a0a9951-04b0-4526-9718-c7c72d24e025.png)
(I var edited the lower blast door to have `dir = 4`, which is east-facing. The upper blast door starts the round facing the wrong way)

**Comment by LemonInTheDark** [comment]
if that's icebox it's likely a won't fix for now, for similar reasons.
The "intent" was to only run pubby, to avoid massive conflicts and lessen the load on mappers until we prep for a full merge.

Unfortunately mothblocks betrayed me. Fixing now

**Comment by PKPenguin321** [comment]
This is not entirely a mapping issue after all: Airlocks placed by players seem to always be south-facing with the RCD.
![image](https://user-images.githubusercontent.com/8345184/167279973-d6d729d5-f2de-493a-aa0b-8c2322f4f973.png)

**Comment by LemonInTheDark** [comment]
That part, that part's on me.
We need a way to reorient airlocks as a part of their build step, and with the rcd

**Comment by PKPenguin321** [comment]
I don't think it should even be possible to have an incorrectly oriented airlock. It should probably be handled automatically through code by checking what walls are around the airlock

**Comment by LemonInTheDark** [comment]
I tried that, it's not always possible to get it right, particularly when applied to similar objects, like shutters or blast doors.
Might be ok for just airlocks, but that's just kicking the can down the road. Need a proper solution.
